Makefile

```diff
@@ -21,6 +21,7 @@ ACCEPTANCE_RUN_TESTS=.
 PKG         := ./...
 TAGS        :=
 TESTS       := .
+TESTARGS    := -shuffle=on -count=3
```

Comment:
Based on discussion recently on this topic, I think count should be 1 by default and 2 in CI

Use TESTFLAGS below instead